orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -34,6 +34,8 @@ var (
 )
 
 func TestSpecServicer_NewSpecServicerFromFile(t *testing.T) {
+	os.RemoveAll(tmpDir)
```

Comment:
For code clarify: group the RemoveAll calls at the top. First this one, followed immediately by the deferred call